gobblin-utility/src/main/java/gobblin/util/AvroUtils.java

```diff
@@ -67,6 +68,7 @@
   private static final String FIELD_LOCATION_DELIMITER = ".";
 
   private static final String AVRO_SUFFIX = ".avro";
+  public static final String SPACE_SLASH_REGEX = "[\\s/]";
```

Comment:
HDFS does not allow ':' in file names. I'd also put '\' in the blacklisted characters.